Readme.md

```diff
@@ -87,6 +85,9 @@ new Vue({
   @Prop({ default: () => [] as ICategory[] }) exceptCategories!: ICategory[];
   @Prop({ default: () => [] as Emoji[] }) exceptEmojis!: IEmoji[];
   @Prop({}) i18n!: Object;
+  @Prop({}) layout?: 'flex' | 'grid';
+  @Prop({}) width?: String | number;
+  @Prop({}) height?: String | number;
```

Comment:
This is properties of styles, has conflict if the use set changes on property style of component